encoding/thrift/register.go

```diff
@@ -59,6 +59,6 @@ func Register(registry transport.Registry, service Service) {
 	proto := service.Protocol()
 	for method, h := range service.Handlers() {
 		handler := thriftHandler{Handler: h, Protocol: proto}
-		registry.Register(procedureName(name, method), handler)
+		registry.Register("", procedureName(name, method), handler)
```

Comment:
Yeah, we'll have to figure out an API for when users do want to override the service name. Usually, they shouldn't need to.